### PR TITLE
Add Galadriel devnet

### PR DESCRIPTION
**Pull request by madschristensen99**
### Description
Add Galadriel Devnet

### Backward compatibility

Yes

### CODE REVIEW

File: chains/galadrieldevnet/logo.svg
Comment (jmrossy) [inline chains/galadrieldevnet/logo.svg:1]: This is not a valid SVG image file. Did you use a PNG or something like that? The logos must be vector graphics.

**Review by jmrossy — COMMENTED** [review]
Mostly looks good but you need a changeset and the logo is not valid

File: chains/galadrieldevnet/logo.svg
Comment (madschristensen99) [reply chains/galadrieldevnet/logo.svg:1]: Yes sorry, I will adjust that file. I will make the changeset as well.

**Comment by madschristensen99** [comment]
I have made the requested change to the logo, and I believe there is now a changeset. So we should be good to go with this merge.

**Review by jmrossy — APPROVED** [review]

**Pull request merged**